### DISCUSSION TITLE
Aplós - A sleek VitePress theme

### DISCUSSION THREAD

**Discussion by GabsEdits** [vuejs/vitepress#3647] [Show and tell]
Hi VitePress Community! I am here to show a recent project I've been working on: Aplós - A sleek, lightweight, contemporary, and purposefully designed VitePress theme & Static Template. I've worked on it to make it as simple and nice as I can. You can view both the demo and the project on https://aplos.gxbs.me.

[![Screenshot of Aplos](https://github.com/GabsEdits/aplos/assets/110247388/3c98f962-b560-442a-9f56-9b805156a272)](https://aplos.gxbs.me/)

**Comment by GabsEdits** [comment]
I've managed to bundle it into an NPM Package, you can the find the package here: https://npmjs.com/package/aplos :smile:

**Comment by xiyuvi** [comment]
cannot access

**Reply by brc-dd** [reply]
https://github.com/aplosdev/aplos
https://aplos.gxbs.dev/
